digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/service/VipsRestServiceImpl.java

```diff
@@ -45,7 +45,7 @@ public GetCodetablesServiceResponse getCodeTableValues(String correlationId) {
                 .headers(headers -> headers.setBasicAuth(properties.getVipsRestApiUsername(), properties.getVipsRestApiPassword()))
                 .retrieve()
                 .bodyToMono(GetCodetablesServiceResponse.class)
-                .doOnSuccess(System.out::println)
+                //.doOnSuccess(System.out::println)
```

Comment:
I think we should change this to log both success and failure cases at information level with the current logger that we use at some point.

Reply:
Agreed. I commented out this line as it was clogging up the Splunk output. Every time the code tables is requested, we get lines and lines of output in Splunk.